src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php

```diff
@@ -66,13 +66,22 @@ private function __construct(array $thresholds = [], $regex = '', $verboseOutput
                 'self' => $this->thresholds['indirect'],
             ];
         }
-        foreach ($groups as $group) {
+        foreach ($groupsWithAThreshold as $group) {
             if (!isset($this->thresholds[$group])) {
                 $this->thresholds[$group] = 999999;
             }
         }
         $this->regex = $regex;
-        $this->verboseOutput = $verboseOutput;
+
+        $groupsWithAVerbosity = ['unsilenced', 'direct', 'indirect', 'self', 'other'];
+        $this->verboseOutput = array_fill_keys($groupsWithAVerbosity, false);
```

Comment:
should be replaced with a static array, no need for the `$groupsWithAVerbosity` variable

Reply:
Are you sure? That variable is used 2 lines below that.